<launch>

	<!-- Elektron node-->
	<include file="$(find elektron_bringup)/launch/elektron_LMS100.launch" />


	<!-- Joy node -->
	<node respawn="true" pkg="joy" type="joy" name="joy" >
		<param name="dev" type="string" value="/dev/input/js0" />
	</node>

	<!-- Axes and scaling -->
	<param name="axis_linear" value="1" type="int"/>
	<param name="axis_angular" value="0" type="int"/>
	<param name="scale_linear" value="1.0" type="double"/>
	<param name="scale_angular" value="0.23" type="double"/>
	
	<!-- At last - teleoperation node -->
	<node pkg="elektron_teleop" type="elektro_teleop_joy" name="teleop"/>

</launch>